src/rainflow.py

```diff
@@ -106,7 +106,7 @@ def extract_cycles(series, left=False, right=False):
             elif len(points) == 3:
                 # Y contains the starting point
                 # Count Y as one-half cycle and discard the first point
-                yield points[-3], points[-2], 0.5
+                yield points[0], points[1], 0.5
```

Comment:
Just readability - My opinion is that its easier to read points[0] and points[1] when you know the length of the array is 3 instead of using points[-3] and points[-2]. 

I'm removing this change if wanted.

Reply:
I agree, this is more readable.